src/ontodia/editor/editorController.tsx

```diff
@@ -893,6 +958,87 @@ function placeElements(
     return elements;
 }
 
+function placeElementsAround(params: {
+    view: DiagramView;
+    elementIris: ReadonlyArray<ElementIri>;
+    zeroPosition: Vector;
+    startAngle?: number;
+    callBack?: (elements: Element[]) => void;
+}) {
+    const {
+        elementIris,
+        view,
+        zeroPosition,
+        startAngle = 0,
+        callBack,
+    } = params;
+
+    const prefferedLinksLength = 300;
+    const elements = elementIris.map(iri => view.model.createElement(iri));
+
+    const listener = new EventObserver();
+    listener.listen(view.model.events, 'changeCells', () => {
```

Comment:
Wait until all elements have not zero size, then place them around on a half-circle radius.